.github/BOTMETA.yml

```diff
@@ -1480,7 +1482,7 @@ macros:
   team_ftd: annikulin
   team_gitlab: dj-wasabi Lunik marwatk Shaps waheedi
   team_google: rambleraptor erjohnso
-  team_hcloud: cschmitt-hcloud LKaemmerling
+  team_hcloud: cschmitt-hcloud LKaemmerling resmo
```

Comment:
I am not actively maintaining hcloud, just helped out with "onboarding"

Reply:
Ok, it's because you were listed as reviewer on the community wiki.
(Which would imply being notified and having shipit rights)
I'll remove your name.